### DISCUSSION TITLE
How can I start from beginning of podcast?

### DISCUSSION THREAD

**Discussion by plum** [tramhao/termusic#374] [Q&A]
Is it possible to start from the beginning of a podcast which has prgressed quite far? 

One scenario is simply to replay a podcast that I ahve been listened to. 

A second scenario  is that frequently on a 'new'  podcast which I have not listened to ,  I am presented with a  progress bar may be advanced as much as  1 hour of 1 hour and 10 mitues , for example!  

So you see,  skipping backward by 10 second, s at a time wont always  do the trick for me.

**Comment by tramhao** [comment]
By default, for music, the play position will not be remembered, but for podcasts it'll be remembered. This can be changed in config file server.toml. Please check if it works for you.

**Comment by hasezoey** [comment]
> A second scenario is that frequently on a 'new' podcast which I have not listened to , I am presented with a progress bar may be advanced as much as 1 hour of 1 hour and 10 mitues , for example!

this sounds like a bug, please report it as a issue (assuming it is actually new and you didnt silently let it run)

> So you see, skipping backward by 10 second, s at a time wont always do the trick for me.

to my knowledge there is currently no "seek to beginning" shortcut, you could make a new suggestion issue for that.
